web/src/click_handlers.ts

```diff
@@ -730,6 +730,51 @@ export function initialize(): void {
         browser_history.go_to_location(target);
     });
 
+    $("body").on("click", ".formatting-control-scroller-button", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const scroller_button_width_px = 48;
+        // We scroll 60% of the viewable area on each click
```

Comment:
I think this comment would be a lot nicer if it explained the reasoning for 60% as the number here.

I kinda suspect we might want something more like (80% - 2 * scroller_button_width_px), basically because the pre-scroll and post-scroll corners were covered by the scroller widget, so then the actual visible buttons would have 20% overlap? Might need to be a bit more complex based on how many scroller widgets were visible before and after.

Reply:
For sure. I figured that would be a value to be tinkered with based on some real-world usage; 60% just gets in the ballpark.

Reply:
I opted for something similar, but just reduced the overall viewable area by 2 * scroller_button_width--rather than the percentage. I think the result is an improvement.